.travis.yml

```diff
@@ -36,10 +35,15 @@ matrix:
   include:
   - python: 3.7
     env: JOBQUEUE="slurm"
-  allow_failures:
+  - language: go
+    go: 1.13
+    env:
+      - JOBQUEUE="singularity"
+      - INSTALL_TYPE="develop"
   - python: 3.7
     env: INSTALL_DEPENDS="pip==10.0.1 setuptools==30.3.0"
-  - python: 3.8
```

Comment:
any reason to drop this?

Reply:
not really, just wanted removed something. 3.8 is removed wit pip==10.0.1 only, there are still 7 other runs with 3.8

Reply:
thank you.